_data/trans/gr.json

```diff
@@ -0,0 +1,41 @@
+{	
+	"banner": “Δοκιμάστε το νέο μας <span>Chrome Extension</span>",
```

Comment:
This line, and several others, seem to have "fancy quotes", like this: `“`. Please substitute these with normal quotes, like this: `"`.